### PR TITLE
Update README to be more descriptive

### PR DESCRIPTION
**Pull request by crombach**
Resolves issue #39.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+- C++11 Compiler
+- Cmake
+- Make
+
+### Instructions
 
 ```bash
 git clone --recursive https://github.com/tz0/game
 cd game
+git submodule init && git submodule update
```
Comment (narrative) [inline README.md]: this step is not necessary if it was cloned with the --recursive option.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,26 @@
-# game
-For game project
+# Jetpack Game
 
-## Building
+This game is being build for Professor Pieter Peers' Introduction to Video Game Design (CSCI437) class at The College of William & Mary.
```
Comment (narrative) [inline README.md]: ..game is being buil**t** for..

Comment (crombach) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (crombach) [reply README.md]: fixed

**Pull request merged**